packages/driver/src/cy/commands/sessions/index.ts

```diff
@@ -57,7 +57,7 @@ export default function (Commands, Cypress, cy) {
       }
 
       // stringify deterministically if we were given an object
-      id = _.isString(id) ? id : stringifyStable(id)
+      id = _.isString(id) ? id : stringifyStable(id) as string
```

Comment:
Something went haywire with bumping the sub-dependency of json-stable-stringify, I imagine something with how lerna works. Adding these types fixed it and I'm not concerned about if we're using the newer minor version of this sub dep.